prow/cmd/gerrit/README.md

```diff
@@ -29,4 +29,4 @@ it empty for anonymous access to gerrit API.
 
 Also take a look at [gerrit related packages](/prow/gerrit/README.md) for implementation details.
 
-you might also want to deploy [Crier](/prow/cmd/crier) which handles report presubmit jobs back to gerrit.
+You might also want to deploy [Crier](/prow/cmd/crier) which reports job results back to gerrit.
```

Comment:
also update the description on line 3-5 to include postsubmits?

Reply:
:+1: